timemachine/cpp/src/nonbonded_interaction_group.cu

```diff
@@ -220,11 +237,16 @@ void NonbondedInteractionGroup<RealType>::execute_device(
 
     const int tpb = warp_size;
     const int B = ceil_divide(N_, tpb);
+    const int K_B = ceil_divide(K_, tpb);
```

Comment:
nit: the naming of `K_B` seems a bit confusing to me. It's really a number of blocks, right? (would `B_K` make more sense?)